src/course-unit/move-modal/moveModal.test.tsx

```diff
@@ -0,0 +1,191 @@
+import MockAdapter from 'axios-mock-adapter';
+import { render, waitFor, within } from '@testing-library/react';
+import { AppProvider } from '@edx/frontend-platform/react';
+import { IntlProvider } from '@edx/frontend-platform/i18n';
+import { initializeMockApp } from '@edx/frontend-platform';
+import { getAuthenticatedHttpClient } from '@edx/frontend-platform/auth';
+import { Store } from 'redux';
+
+import userEvent from '@testing-library/user-event';
+import initializeStore from '../../store';
+import { getOutlineInfo } from '../data/api';
+import { courseOutlineInfoMock } from '../__mocks__';
+import { executeThunk } from '../../utils';
+import { getCourseOutlineInfoQuery } from '../data/thunk';
+import { IframeProvider } from '../context/iFrameContext';
+import MoveModal from './index';
+import messages from './messages';
+
+interface CourseOutlineChildInfo {
```

Comment:
[question]: Just a clarification, can we use these TS types inside the component or maybe we are already using them?

Reply:
Reused from the component